### DISCUSSION TITLE
new config location?

### DISCUSSION THREAD

**Discussion by Tealk** [FreshRSS/FreshRSS#4431] [Q&A] [answered]
Hello,

is there a new config location?
My instance is running although there is no config in the data folder.
![image](https://user-images.githubusercontent.com/12276250/176855899-89f70345-2359-4d59-aea5-6a9ee2bee9be.png)

**Comment by Alkarex** [comment, marked as answer]
Hello,
`./data/config.php` is the right place, but watch out if you are using for instance Docker volumes, in which case it might be in that volume.
How did you install FreshRSS?

**Reply by Tealk** [reply]
Similar to [this tutorial](https://freshrss.github.io/FreshRSS/en/admins/06_LinuxInstall.html), only under Debian11 and with nginx.
I moved to a new server and actually reinstalled everything and only took the database with me, so I'm surprised the config is missing.

**Reply by Alkarex** [reply]
When migrating server, it is best to take the full `./data/` directory (plus databases if not using SQLite)

**Reply by Alkarex** [reply]
`./data/config.php` is created during the install process (either through the Web installer, or the command line installer)

**Reply by Tealk** [reply]
well the old server no longer exists, I'm just wondering how that can run without config. Where do the mariadb login data come from?

**Reply by Tealk** [reply]
ok my mistake sorry
i had a folder FreshRSS in /var/www and there the config was missing; but in the tutorial it is installed in /usr/share
